### PR TITLE
Bors-ng: Add documentation and status-icon

### PR DESCRIPTION
**Pull request by cevich**
***Note:*** Depends on #2115

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -116,6 +117,35 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
...
+
+Maintainers should never merge anything directly into upstream
+branches.  Instead, interact with the [bors-ng bot](https://bors.tech/)
+through PR comments as summarized below. This ensures all upstream
+branches contain commits in a predictable order, and that every commit
+has passed automated testing at some point in the past. A
+[Maintainer portal](https://app.bors.tech/repositories/22803)
+is available, showing all PRs awaiting review and approval.
+
+Common [bors-ng comment commands](https://bors.tech/documentation/):
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: These commands are to be added by themselves in a line in a comment of the PR?  Do you want to state something to that effect?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -116,6 +117,35 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
...
+
+* `bors ping` - Confirm bot is functioning, it will post a comment in the PR if so.
+* `bors try` - Optional / simulate merging of one (or more) PRs, executing automated
+  tests on the combination, and reporting results.  Good for validating `bors.toml`
+* `bors r+` - Mark the PR has reviewed.  Once a PR receives the required number of
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: 'has reviewed' -> 'as reviewed'

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -116,6 +117,35 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
...
+  approvals (set in `bors.toml`), the PR will go into the merge queue for testing
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: what's an approval?  'LGTM', 'bors r+', or something else?

File: CONTRIBUTING.md (hunk 1 above)
Comment (cevich) [reply CONTRIBUTING.md]: good idea.  Got it.

File: CONTRIBUTING.md (hunk 2 above)
Comment (cevich) [reply CONTRIBUTING.md]: Got it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (cevich) [reply CONTRIBUTING.md]: Hmmm, ya that's confusing, I will clarify...

**Comment by cevich** [comment]
> Really nifty work overall @cevich. I'm just shocked you've any hair left after this process...

Lol thanks, believe me, it's starting to thin :disappointed: Hopefully it will all work out, and maybe, just maybe, after some runtime battle-testing, we can switch to using bors-ng in our other repos.

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
bors merge

**Comment by cevich** [comment]
Updated with screen-shot showing how to interact with a bors-ng 'try' or 'merge' in progress.

**Comment by cevich** [comment]
(That failure was my doing, travis was doing bad things.  We need to wait for #2115 to merge, then bors will automatically retry and it should work)

**Comment by cevich** [comment]
bors retry

**Comment by rhatdan** [comment]
@cevich Rebase the image.

**Comment by cevich** [comment]
bors retry

**Comment by cevich** [comment]
hehehe...it was worth a try :grinning:

**Comment by rhatdan** [comment]
:+1: 
bors r+

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @mheon @nalind @QiWang19 PTAL

**Comment by TomSweeneyRedHat** [comment]
Going to add @baude as an fyi

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -116,6 +117,57 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
...
+has passed automated testing at some point in the past. A
+[Maintainer portal](https://app.bors.tech/repositories/22803)
+is available, showing all PRs awaiting review and approval.
+
+### Common [bors-ng comment commands](https://bors.tech/documentation/):
+
+(must be on a single comment-line, without any other extraneous text)
+
+* `bors r+` - Mark the PR as reviewed, and increment the approval number by one.
+  When minimum number of approvals are met, the bors bot will add the PR into a
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: nit "When minimum" -> "When the minimum"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -116,6 +117,57 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
...
+* `bors try` - Optional / simulate the actions of `bors r+` (see above) having
+  met the minimum number of required approvals.  The result will be reported
+  back as a comment in the PR, by the bors bot.
+* `bors ping` - Confirm bot is functioning, it will post a comment in the PR if so.
+
+
+### Interaction/Monitoring
+
+Bors-ng relies on the regular branch-testing occurring when it updates the
+special branches ('trying' or 'staging').  Therefor you may use the full
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: "Therefor" -> "Therefore"

**Comment by cevich** [comment]
Woopsie.  @TomSweeneyRedHat @rhatdan I misinterpreted a setting - fixed it and the documentation in the most recent commit.

Saying 'bors r+' alone should work now (no github reviewers required) - unless you want them to be required, then I can set it back to `1` :grin:

**Comment by rhatdan** [comment]
@ceve PR Status?

**Comment by rhatdan** [comment]
@cevich PR STatus?

**Comment by rhatdan** [comment]
Bors does not like me today.  Forcing merge.

**Comment by cevich** [comment]
> Waiting for PR status (Github check) to be set, probably by CI. Bors will automatically try to run when all required PR statuses are set.

Hrmmm, not sure about that one.  I'll need to check on the forum if it keeps happening.

**Pull request merged**